keps/sig-network/2079-network-policy-port-range/README.md

```diff
@@ -203,7 +221,7 @@ validation should be done by CNIs.
 with generally positive feedback on its usage.
 - Feature Gate is enabled by Default.
 
-#### GA Graduation
+#### GA
```

Comment:
Has this been met? Can you name the CNI providers that support it?

Reply:
Added